src/SWP/Bundle/ContentBundle/Tests/Controller/RoutesControllerTest.php

```diff
@@ -63,6 +63,26 @@ public function testCreateContentRoutesApi()
         $this->assertEquals('{"id":"\/swp\/default\/routes\/simple-test-route","content":null,"static_prefix":null,"variable_pattern":null,"name":"simple-test-route","children":[],"id_prefix":"\/swp\/default\/routes","_links":{"self":{"href":"\/api\/v1\/content\/routes\/\/simple-test-route"}}}', $client->getResponse()->getContent());
     }
 
+    public function testCreateContentRoutesApi()
```

Comment:
Static page is misleading - as those routes can be host really dynamic pages (depends from template). 

We have two type of routes (collections and content). Collection can host many articles under it, content can have just one or none - so it can be for example used for section/issue/topic/event page.